File: golang/3237.go
func findNumbers(nums []int) int {
    var eventick int = 0
    for i := 0; i < len(nums); i++ {
        var numcount int = 0
        for nums[i] != 0 {
            nums[i] /= 10
            numcount++
        }
        if numcount%2 == 0 {
           eventick++
        }
    }
    return eventick
}

File: rust/3237.rs
impl Solution {
    pub fn find_numbers(nums: Vec<i32>) -> i32 {
        let mut eventick = 0;
        for mut number in nums.into_iter() {
            let mut numcount = 0;
            while number != 0 {
                number /= 10;
                numcount += 1;
            }
            if numcount%2 == 0 {
                eventick += 1;
            }
        }
        return eventick; 
    }
}
